gambling.js

```diff
@@ -45,6 +80,44 @@ exports.dischargeScrubBubble = function (userID, botSpam) {
     const title = dropped + ' Scrubbing ' + msg + ' arrived for duty!';
     util.sendEmbedMessage(title, null, c.BUBBLE_IMAGES[droppedImg-1]);
 }
+/**
```

Comment:
add description of function above @params as seen above in various other function's JSDoc